lib/shared/config/loadfiles.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var get = require('lodash.get');
+var set = require('lodash.set');
+var isString = require('lodash.isstring');
+var configSpecs = require('./cfg-specs');
+
+function loadConfigFiles(configFilesBase, keysInOrder) {
+  var config = {};
+
+  var configFilePaths = keysInOrder.map(function(key) {
```

Comment:
If config files are found in user home directory and project directory, both file paths are given by `fined`. In that case each config in both project dir should be given priority over each config in user home dir, I think.

Though this order might be determined by descriptions at `Liftoff` constructor, I want to specify it explicitly.